sysprep/instance_setup.ps1

```diff
@@ -287,6 +305,7 @@ else {
   Write-Log 'Running startup scripts from metadata server.'
   Create-GCEStartup
   Invoke-ExternalCommand schtasks /run /tn GCEStartup
+  Optimize-PowershellAssemblies
```

Comment:
Whats' the time for this to run, since this isn't required for instance setup it would make more sense to add before Wait-Job (if its <2s) or after if greater. you could also start it in the background if that makes more sense.

Reply:
On an n1-standard-4 I've seen this take around 30s. I've moved this down to just after Wait-Job - is that what you meant?